scripts/file.pl

```diff
@@ -19,7 +20,7 @@
 -echo abuses a bug in the script and is useful for testing
```

Comment:
Either remove this note (but I find -echo useful and had forgotten it relies on side effects) or consider generically treating unknown options as commands...